File: C#/02. Simple Operations and Calculations/Exercise/Charity Campaign.cs
using System;

namespace Charity_Campaign
{
    class Program
    {
        static void Main(string[] args)
        {
            int days = int.Parse(Console.ReadLine());
            int bakers = int.Parse(Console.ReadLine());
            int cakes = int.Parse(Console.ReadLine());
            int waffles = int.Parse(Console.ReadLine());
            int pancakes = int.Parse(Console.ReadLine());

            double cakePrice = 45 * cakes;
            double wafflePrice = 5.8 * waffles;
            double pancakePrice = 3.2 * pancakes;
            double total = (cakePrice + wafflePrice + pancakePrice) * bakers * days;

            Console.WriteLine($"{total - total / 8:F2}");
        }
    }
}

File: Python/02. Simple Operations and Calculations/Exercise/Charity Campaign.py
days = int(input())
bakers = int(input())
cakes = int(input())
waffles = int(input())
pancakes = int(input())

cakePrice = 45 * cakes
wafflePrice = 5.8 * waffles
pancakePrice = 3.2 * pancakes
total = (cakePrice + wafflePrice + pancakePrice) * bakers * days * (7 / 8)

print(f"{total:.2f}")